apps/dashboard-ui/src/app/components/long-card/long-card.component.html

```diff
@@ -0,0 +1,9 @@
+@for (card of cardData(); track card.label) {
+  <a [routerLink]="['/', card.route]" class="long-card" rel="noopener noreferrer">
```

Comment:
Make the initial `/` part of the `card.route` data instead of prepending it here